dev/docker-compose.yml

```diff
@@ -27,10 +27,20 @@ services:
       PRIVKEY_FILE: /run/secrets/dummy.pem
       BIND_ADDR: 0.0.0.0:8080
       RUST_LOG: ${RUST_LOG}
+      ALL_PROXY: http://mitmproxy:8080
     secrets:
       - pki.secret
       - dummy.pem
       - root.crt.pem
+  mitmproxy:
+    image: mitmproxy/mitmproxy
+    stop_signal: SIGKILL
+    volumes:
+      - ./mitmproxy:/home/mitmproxy/.mitmproxy
```

Comment:
```suggestion
```

For now, we can leave this out since we don't need TLS.

(The .gitignore entry does not hurt, though)